xds/src/main/java/io/grpc/xds/GoogleCloudToProdNameResolver.java

```diff
@@ -169,9 +173,9 @@ public void run() {
     executor.execute(new Resolve());
   }
 
-  private static ImmutableMap<String, ?> generateBootstrap(String zone, boolean supportIpv6) {
+  private ImmutableMap<String, ?> generateBootstrap(String zone, boolean supportIpv6) {
     ImmutableMap.Builder<String, Object> nodeBuilder = ImmutableMap.builder();
-    nodeBuilder.put("id", "C2P");
+    nodeBuilder.put("id", "C2P-" + rand.nextInt());
```

Comment:
I know we agreed to just add a random number but it will be nice to add some node identification info such as what you get from `http://metadata.google.internal/computeMetadata/v1/instance/network-interfaces/0/ip` or `http://metadata.google.internal/computeMetadata/v1/instance/id` *and plus* the random number

Reply:
It might be useful (?) for debugging from TD's perspective. Not sure if TD is already able to identify client VM from other metadata information. What about just bringing this up to @markdroth?

Reply:
Agreed,  this has to be uniform across all gRPC languages

Reply:
Discussion in b/183237952.